crates/rspack_core/src/compiler/compilation.rs

```diff
@@ -1437,19 +1437,22 @@ impl Compilation {
         .compilation_hooks
         .optimize_modules
         .call(self)
-        .instrument(info_span!("Compilation:optimize_modules"))
+        .instrument(info_span!("hook:optimize_modules"))
```

Comment:
We had tracing for every plugin hook written in macro https://github.com/web-infra-dev/rspack/blob/5a49a0169c7ae795ffa06cddf648ea48ce23c1c7/crates/rspack_macros/src/plugin.rs#L186-L193
you can add tracing at the macro so you don't need to add it at every callsite

Reply:
this comment out macro should be added back

Reply:
will refactor all instrument using macro in following pr